FinniversKit/Sources/Components/TransactionEntryView/TransactionStepView.swift

```diff
@@ -17,7 +17,7 @@ public class TransactionStepView: UIView {
 
     private lazy var textLabel: Label = {
         let label = Label(style: .detail, withAutoLayout: true)
-        label.numberOfLines = 2
+        label.numberOfLines = 5
```

Comment:
Good catch 😄 
We decided to have a maximum of two lines on the front page entry, just because that fits the designs best.
I suggest we can make it possible to configure differently, but keep 0 as the default, like this:
```
public func configure(withTitle title: String?, text: String?, showWarningIcon: Bool, numberOfTextLines: Int = 0) {
         textLabel.numberOfLines = numberOfTextLines
```
And from `TransactionEntryView`, we can keep `numberOfTextLines: 2`
Or what do you think? 😄

Reply:
Sure, i'll fix it 👍